pkg/kubelet/util/format/pod.go

```diff
@@ -33,12 +34,28 @@ func Pod(pod *api.Pod) string {
 	return fmt.Sprintf("%s_%s(%s)", pod.Name, pod.Namespace, pod.UID)
 }
 
+// PodWithDeletionTimestamp is the same as Pod. In addition, it prints the
+// deletion timestamp of the pod if it's not nil.
+func PodWithDeletionTimestamp(pod *api.Pod) string {
+	var deletionTimestamp string
+	if pod.DeletionTimestamp != nil {
+		deletionTimestamp = ":DeletionTimestamp=" + pod.DeletionTimestamp.UTC().Format(time.RFC3339)
+	}
+	return Pod(pod) + deletionTimestamp
+}
+
 // Pods returns a string representating a list of pods in a human
 // readable format.
 func Pods(pods []*api.Pod) string {
 	return aggregatePods(pods, Pod)
 }
 
+// PodsWithDeletiontimestamp is the same as Pods. In addition, it prints the
+// deletion timestamp of the pods if they are not nil.
```

Comment:
I think we should add a new update type DELETE to make this clear. For now, logging the timestamps is fine.